swagger.json

```diff
@@ -5796,6 +5861,55 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "Contact": {
+            "type": "object",
+            "description": "Information about a contact.",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "ID of the contact",
+                    "format": "int64",
+                    "example": 123
+                },
+                "userId": {
+                    "type": "integer",
+                    "description": "User ID of the contact",
+                    "format": "int64",
+                    "example": 123
+                },
+                "orgId": {
+                    "type": "integer",
+                    "description": "Organization ID of the contact",
+                    "format": "int64",
+                    "example": 123
+                },
+                "firstName": {
+                    "type": "string",
+                    "description": "First name of the contact",
+                    "example": "Jane"
+                },
+                "lastName": {
+                    "type": "string",
+                    "description": "Last name of the contact",
+                    "example": "Jones"
+                },
+                "email": {
+                    "type": "string",
+                    "description": "Email address of the contact",
+                    "example": "jane.jones@yahoo.com"
+                },
+                "phone": {
+                    "type": "string",
+                    "description": "Phone number of the contact",
+                    "example": "111-222-3344"
+                },
+                "webhook": {
+                    "type": "string",
+                    "description": "Webhook associated with the contact",
+                    "example": "???"
```

Comment:
looks like we need an example here

Reply:
@silverspace any recommendations on whether we should return webhooks here, and what an example webhook looks like?

Reply:
I think this is an unused field that we should remove from the contacts table.

Reply:
Removed the field.